css-layout-classes/style.css

```diff
@@ -0,0 +1,66 @@
+/* A new css rule set selecting all elements in the 'body' elements ... Opening curly bracket */
+body {
+  margin: 0 auto; /* centering body */
+}
+
+/* A new css rule set selecting all elements in the 'body' elements and 'container' class elements ... Opening curly bracket */
+body,
+.container {
+  max-width: 75vh;
+  min-height: 100vh;
+}
+
+/* A new css rule set selecting all elements in 'container' class elements ... Opening curly bracket */
+.container {
+  min-height: 100%; /* Set container to take up the full viewport height */
+}
+
+/* A new css rule set selecting all elements in 'img' elements ... Opening curly bracket */
+img {
+  width: 100%;
+}
+
+/* A new css rule set selecting all elements in 'h1' element and 'h2' element ... Opening curly bracket */
+h1,
+h2 {
+  font-family: Arial, sans-serif;
+  color: black;
+}
+
+/* A new css rule set selecting all elements in the 'p' element ... Opening curly bracket */
+p {
+  font-family: Arial, sans-serif;
+  color: black;
+  font-size: 0.89rem;
+}
+
+/* A new css rule set selecting all elements in the 'row' class elements ... Opening curly bracket */
+.row {
+  display: flex;
+  max-height: fit-content;
+  justify-content: center;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+/* A new css rule set selecting all elements in the 'column-full' class elements ... Opening curly bracket */
+.column-full {
+  flex-direction: column;
```

Comment:
`flex-direction` is a parent property, so it gets applied where `display: flex` gets applied. Since this is a flex child, this has no impact, so it should be removed